### PR TITLE
Opennebula support

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

OpenNebula support for the big glue 2.1 rebase ⚠️

### CODE REVIEW

**Comment by gwarf** [comment]
I've rebased the glue_2.1_branch with the latest master from upstream and unfortunately, it seems you will have to rebase and merge some conflicts (or delete and restart from up-to-date branch), sorry. Hope I didn't messed up and lost some work :/

**Comment by enolfc** [comment]
This last change makes it to generate a valid render, but it still misses (at least) getting extra information from the images.

**Comment by gwarf** [comment]
So you can now rebase once again, thanks!

**Comment by enolfc** [comment]
Ready for reviews!

Tested @ CESGA and @ EBI with both old Glue and Glue21

File: Jenkinsfile [hunk 1]
```diff
@@ -29,7 +29,19 @@ pipeline {
                 sh 'tox -e cover'
 
                 echo 'Generating Cobertura report..'
-                sh 'tox -e cobertura'
+                writeFile file: 'tox.ini.cobertura', text: '''[tox]
```
Comment (gwarf) [inline Jenkinsfile]: Shouldn't be there, it was moved to tox.ini

File: cloud_info_provider/providers/static.py [hunk 2]
```diff
@@ -155,6 +155,14 @@ def get_compute_shares(self, **kwargs):
                                 None,
                                 fields,
                                 prefix='')
+        # if there is no explicit membership defined, assume VO:<nameofshare>
```
Comment (gwarf) [inline cloud_info_provider/providers/static.py]: OK, nice!
Might be worth documenting this in sample files.

**Comment by enolfc** [comment]
We keep it, but the default should be good already (project = vo name),
it's also commented a few lines above

File: cloud_info_provider/providers/static.py (hunk 2 above)
Comment (enolfc) [reply cloud_info_provider/providers/static.py]: It's documented, see https://github.com/gwarf/cloud-info-provider/pull/8/files/6e5277453e8f273720f1b8b3e0f01a87d4010974#diff-9247b136d1ee94dde463b142bd2a873fR80
and https://github.com/gwarf/cloud-info-provider/pull/8/files/6e5277453e8f273720f1b8b3e0f01a87d4010974#diff-526b6d8c3f7e68d8fd1df2fb5105179fR98

File: Jenkinsfile (hunk 1 above)
Comment (enolfc) [reply Jenkinsfile]: too many rebases :(, will fix this

File: etc/sample.opennebula.yaml [hunk 3]
```diff
@@ -85,27 +85,27 @@ compute:
...
+            #sla: UNKNOWN
             # Type of network: infiniband, gigabiteethernet...
             network_info: UNKNOWN
-            membership:
-                - VO:ops
+            # Default membership is VO:<share name>, specify if finer grained AuthZ is used
+            #membership:
+            #    - VO:ops
         fedcloud.egi.eu:
-            project: fedcloud
```
[the remark this answers is not in the document]
Comment (enolfc) [reply etc/sample.opennebula.yaml]: We keep it, but the default should be good already (project = vo name), it's also commented a few lines above

(comment sent by email was not nicely included here, so repasting)

**Comment by enolfc** [comment]
@gwarf could you check again? Thanks!

File: cloud_info_provider/providers/static.py (hunk 2 above)
Comment (gwarf) [reply cloud_info_provider/providers/static.py]: Good!

**Pull request merged**